test/api/v3/integration/members/POST-send_private_message.test.js

```diff
@@ -118,4 +118,56 @@ describe('POST /members/send-private-message', () => {
     expect(sendersMessageInReceiversInbox).to.exist;
     expect(sendersMessageInSendersInbox).to.exist;
   });
+
+  it('allows admin to send when sender has blocked to user', async () => {
+    userToSendMessage = await generateUser({
+      'backer.admin': 1,
```

Comment:
`contributor.admin` rather than `backer.admin` :) Also in other places in the tests and code.